demisto_sdk/commands/common/errors.py

```diff
@@ -717,6 +717,14 @@
         "code": "IN157",
         "related_field": "script",
     },
+    "command_reputation_output_is_invalid": {
```

Comment:
Change to something self-explanatory, this is too generic. (an output can be invalid in many ways)